manticore/core/smtlib/visitors.py

```diff
@@ -267,6 +266,7 @@ def __init__(self, **kw):
         BitVecSub: operator.__sub__,
         BitVecMul: operator.__mul__,
         BitVecDiv: operator.__truediv__,
+        BitVecUnsignedDiv: operator.__floordiv__,
```

Comment:
```suggestion
```

This creates a minor conflict with #1530. Nothing breaking, but BitVecUnsignedDiv is defined twice. Other than that, it looks fine to merge to me.

Reply:
So, #1530's `BitVecUnsignedDiv` was not EVM specific?